docs/en/Running-Presto-with-Alluxio.md

```diff
@@ -11,19 +11,19 @@ that you can easily use presto to query Hive tables in Alluxio's tiered storage.
 
 # Prerequisites
 
-The prerequisite for this part is that you have [Java](Java-Setup.html). And the Java version must greater then
-1.8. Alluxio cluster should also be set up in accordance to these guides for either
+The prerequisite for this part is that you have [Java](Java-Setup.html). And the Java version must use Java 8 Update 60 or higher (8u60+), 64-bit.
+Alluxio cluster should also be set up in accordance to these guides for either
 [Local Mode](Running-Alluxio-Locally.html) or [Cluster Mode](Running-Alluxio-on-a-Cluster.html).
 
-Please[Download Presto](https://repo1.maven.org/maven2/com/facebook/presto/presto-server/)。And have finished
-[Hive On Alluxio](http://www.alluxio.org/docs/master/cn/Running-Hive-with-Alluxio.html)
+Please [Download Presto](https://repo1.maven.org/maven2/com/facebook/presto/presto-server/)(This doc use presto-0.160). And have finished
```

Comment:
Hi, yupeng, I just change the guava's version to 18.0, and do not encounter the mailing list problem.